src/ports/catalog/queries.ts

```diff
@@ -642,7 +659,14 @@ export const getCollectionsItemsCatalogQueryWithTrades = (filters: CatalogQueryF
               items.network,
               offchain_orders.open_item_trade_id,
               offchain_orders.open_item_trade_price,
-              LEAST(items.first_listed_at, ROUND(EXTRACT(EPOCH FROM offchain_orders.min_item_created_at))) as first_listed_at,
+              `
+      .append(
+        filters.isOnSale
+          ? SQL`LEAST(items.first_listed_at, ROUND(EXTRACT(EPOCH FROM offchain_orders.min_item_created_at))) as first_listed_at,`
```

Comment:
for some reason, this statement breaks the query in the case asking for ones NOT on sale. Added this logic to avoid sending it (since it won't be used in that context)